File: src/main/kotlin/onl/tesseract/srp/skill/domain/model/crafting/LootCache.java
package onl.tesseract.srp.skill.domain.model.crafting;

import java.util.ArrayList;
import java.util.List;

public record LootCache(
        List<CraftElement> garbage,
        List<CraftElement> done
) {
    public LootCache(){
        this(new ArrayList<>(), new ArrayList<>());
    }
}

File: src/main/kotlin/onl/tesseract/srp/domain/skill/crafting/LootCache.kt
package onl.tesseract.srp.domain.skill.crafting

import onl.tesseract.srp.domain.item.CustomItem

data class LootCache(
    val done: MutableList<CustomItem> = mutableListOf(),
    val garbage: MutableList<CustomItem> = mutableListOf()
    )